home/home_2.2/main.swift

```diff
@@ -0,0 +1,143 @@
+import Foundation
+
```

Comment:
the same concerns related to encapsulation into Image struct and related code refactoring.